.travis.yml

```diff
@@ -6,7 +6,8 @@ rust:
   - nightly
 matrix:
   allow_failures:
-    - rust: nightly
+    - rust: stable
+    - rust: beta
```

Comment:
Hmm that's true. But I think we will notice anyway if `serde_derive` start working on stable :wink: Feel free to merge as is or remove them.